lq1/style_modern.cfg

```diff
@@ -0,0 +1,16 @@
+echo Modern Style
```

Comment:
Can you make these cvars just an alias under our new engine-specific configs? This will keep things more engine-organized and also won't have the side effect of making people think it's broken if they're using some other engine and expecting the behavior to work.

Reply:
That's how I had it at first. I put them in separate config files so that if the user decides that they want to change the way modern style or classic style works, they can just edit those files. Like, maybe they want the classic HUD on modern style because they like the way the classic HUD looks. They can just go into style_modern.cfg and change `cl_sbar` from 0 to 1. With aliases, they would have to completely rewrite those aliases from scratch.

One alternative I considered is having another alias called `lq_style_modern_user` which the user can write themselves and include in autoexec.cfg. And that always gets called after `lq_style_modern_exec`. Then we provide a template autoexec.cfg which includes those alias commands, starting empty.

Reply:
`default_fte.cfg: alias style_modern="exec lq_style_modern_fte.cfg"`
etc. is what I had in mind. If that helps clarify. We just want the multi-engine stuff consolidated down.